api/api-reference.md

```diff
@@ -34,10 +34,10 @@ curl -X GET "https://console.cloud.timescale.com/public/api/v1/projects/{project
 
 You use this endpoint to create and manage the following Tiger Postgres services:
 
-- `TIMESCALEDB`: a Tiger Postgres instance optimized for real-time analytics service For time-stamped data like events,
+- `time-series`: a Tiger Postgres instance optimized for real-time analytics service For time-stamped data like events,
   prices, metrics, sensor readings, or any information that changes over time
-- `POSTGRES`: a vanilla Postgres instance
-- `VECTOR`: a Tiger Postgres instance with vector extensions
+- `none`: a vanilla Postgres instance
+- `ai`: a Tiger Postgres instance with vector extensions
```

Comment:
> You use this endpoint to create and manage the following Tiger Postgres services:

This wording is confusing to me. The list is showing _add-ons_ that can be mixed/matched, not "Tiger Postgres services".

Furthermore, the `addons` request parameter takes an array of values, meaning that you can mix-and-match them (for example, `["time-series"]`, `["ai"]`, and `["time-series", "ai"]` are all valid options). Finally, `none` is not a valid option via the REST API (that's just a CLI thing). To specify no add-ons (i.e. plain Postgres) via the REST API, you just provide an empty list: `[]`.